circuits/src/zk_circuits/valid_wallet_update.rs

```diff
@@ -312,11 +312,6 @@ where
         new_timestamp: Variable,
         cs: &mut PlonkCircuit,
     ) -> Result<(), CircuitError> {
-        // Ensure all orders have boolean side
-        for order in new_wallet.orders.iter() {
-            cs.enforce_bool(order.side)?;
-        }
```

Comment:
Changing the type of `Order::side` to be boolean means this constraint is automatically generated when the type is allocated